kitchen.yml

```diff
@@ -117,7 +168,7 @@ suites:
             - node
             - packages
       pillars_from_files:
-        packages.sls: pillar.example
+        packages.sls: test/salt/pillar/debian.pillar.sls
```

Comment:
This `test/salt` directory is new - not in current template-formula - will this be contributed back to template formula if useful feature?

Reply:
It was suggested in the slack formulas channel a while back and already used in the [apt formula](https://github.com/saltstack-formulas/apt-formula/tree/master/test/salt/pillar), the rationale being something like 'use `pillar.example` if it suits all cases, add different pillars when needed, under a `test/salt/pillars` subdir and states under `test/salt/states`' so they don't clutter the testing directories'

I think adding an example in the `template-formula` would be a nice thing to have.

Reply:
thanks for the updates @javierbertoli